requirements.txt

```diff
@@ -38,6 +38,7 @@ pyusb==1.0.2
 qrcode==6.1
 reportlab==3.5.13
 requests==2.21.0
+setuptools>=v41.6.0
```

Comment:
It doesn't follow the Debian/Ubuntu packages  { 'ubuntu': '39.0.1' , 'debian': '40.8.0'}.
It means that people installing Odoo via the deb package will be flooded with deprecation warnings right ?

Reply:
Well flooded might be a bit strong but yeah.

Reply:
A better solution could be to use more specific warning filters, and ignore the warning from setuptools. I can use the same mechanism to ignore zeep's since we can't do much about these two.